docs/class5/module11/lab4.rst

```diff
@@ -0,0 +1,67 @@
+Lab 11.4: Create User
+-----------------------------------
```

Comment:
make the link matching the title, this will make cloud docs rst builder job failing.

Reply:
same